### PR TITLE
Adding a comment about outdated content

### PR DESCRIPTION
**Pull request by terrimorton**
Testing a blurb we will want to use for content that is no longer being maintained.

### CODE REVIEW

File: iis/develop/using-webmatrix/create-a-website-from-a-gallery-application.md [hunk 1]
```diff
@@ -16,8 +16,12 @@ Create a Website from a Gallery Application
 ====================
 by [Robert McMurray](https://github.com/rmcmurray)
 
+> [!NOTE]
+> This content is outdated and is no longer being maintained. It is provided as a courtesy for individuals who are still using these technologies. This page may contain URLs that were valid when originally published, but now link to sites or pages that no longer exist.
+
 [This is preliminary documentation and is subject to change.]
```
Comment (bariscaglar) [inline iis/develop/using-webmatrix/create-a-website-from-a-gallery-application.md]: In the light of this, would it not make sense to remove the "preliminary documentation" notice?

Comment (terrimorton) [reply iis/develop/using-webmatrix/create-a-website-from-a-gallery-application.md]: Possibly. I was solely tasked with staging the outdated content notice. I have no knowledge of the content on the page and therefore did not feel qualified to change it.

Comment (shirhatti) [reply iis/develop/using-webmatrix/create-a-website-from-a-gallery-application.md]: Agree with @bariscaglar here

**Comment by shirhatti** [comment]
@terrimorton The staging site looks good. Can you expand the scope of the PR to include the entire section.

Let's make sure to centralize the outdated notice in a single snippet file and include it instead of duplicating this notice. See [DocFX file inclusion](https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html?tabs=tabid-1%2Ctabid-a#file-inclusion)

image

**Comment by bariscaglar** [comment]
I have pushed a commit that removes the outdated "preliminary" notice but I realize that @shirhatti has some additional comments.

**Comment by shirhatti** [comment]
I actually noticed this banner on one of the TechNet pages for deprecated content https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/jj635855(v=ws.11)
Could we perhaps look into a banner as such as opposed to what we have now?

**Comment by terrimorton** [comment]
Not sure how that is done. That page is not editable, and I don't see anything in [https://docs.microsoft.com/en-us/contribute/](https://docs.microsoft.com/en-us/contribute/). I directed a question to the @docsmsft Twitter account to try to see how it could be done: [https://twitter.com/TerriMorton/status/956182836998033409](https://twitter.com/TerriMorton/status/956182836998033409)

**Comment by bariscaglar** [comment]
@shirhatti take a look. This only accepts relative links according to the documentation so that's the only part I don't like about it. Otherwise this should work well. https://review.docs.microsoft.com/en-us/IIS/develop/using-webmatrix/create-a-website-from-a-gallery-application?branch=terrimorton-patch-1

**Comment by shirhatti** [comment]
Hmmm. No way to do an xref?

**Comment by shirhatti** [comment]
I'd like to emit a meta tag to prevent indexing by search engines. I couldn't find anything on the docfx docs, so I've created an issue- https://github.com/dotnet/docfx/issues/3579

**Comment by bariscaglar** [comment]
>           Hmmm. No way to do an xref?

Not according to the documentation but let me try just in case they added the feature later but omitted to document it.

**Comment by bariscaglar** [comment]
Unfortunately, no: 
•	Line : [Error] Unable to resolve "[!include[outdatedContent](xref:outdatedContentWarning)] " at line 19: Absolute path "xref:outdatedContentWarning" is not supported. 

It looks like it has to be relative.

**Comment by shirhatti** [comment]
Can you also add this to the yaml header in the doc?

```yml
ROBOTS: NOINDEX,NOFOLLOW
```

**Comment by bariscaglar** [comment]
I also added the ROBOTS: to the YAML header in the included file, although the documentation indicates that this is not supported. So the files that includes this file should have their own YAML header.

YAML header is NOT supported when the file is an inclusion.

**Comment by bariscaglar** [comment]
So I added it to this specific file but that needs to be done for all of them separately.

**Comment by shirhatti** [comment]
> So I added it to this specific file but that needs to be done for all of them separately.

That's unfortunate, but doesn't look like we have other options 😕

**Comment by bariscaglar** [comment]
@shirhatti anything else?

**Comment by shirhatti** [comment]
Nope. Nothing else I can think of

**Pull request merged**